analyzers/tests/SonarAnalyzer.Test/Rules/RouteTemplateShouldNotStartWithSlashTest.cs

```diff
@@ -93,6 +93,7 @@ public class BasicsController : Controller {{(compliant ? string.Empty : " // No
     [DataRow("""[HttpPatch("/IndexPatch")]""")]
     [DataRow("""[HttpHead("/IndexHead")]""")]
     [DataRow("""[HttpOptions("/IndexOptions")]""")]
+    [DataTestMethod]
```

Comment:
We missed that, and as a result the `RouteTemplateShouldNotStartWithSlash_HttpAttributes` test was never executed right?
If so, it seems a very good rule idea: "Methods with DataRow should be marked as DataTestMethod". WDYT, is it worth creating an issue in the backlog?

Reply:
@antonioaversa I'll reverse this as I fixed it here to not pile up irrelevant functional changes https://github.com/SonarSource/sonar-dotnet/pull/9114